downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -42,6 +42,57 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### Improvements                             
 
+#### Create immutable record value using record type with mutable default values
+
+With these improvements, enable the usage of mutable defaults by wrapping them with `cloneReadOnly` when creating an immutable record value with a record type.
+
+```ballerina
+type Temp record {|
+    any[] initalValues = [10, 20, 30];
+|};
+
+function value() {
+    // Now results in compile-time error.
+    Temp & readonly _ = {};
+}
+```
+
+#### Improvements to the usage of default values of record fields
+
+Now, the default value of a record is evaluated only if a value is not provided for the specific field in the mapping constructor.
+
+With these improvements, with record type inclusion, the default value from an included record will not be used if the including record overrides the field.
+
+```ballerina
```

Comment:
Add the example for the simple case. The complex case (inclusion) we can just have the description.

Reply:
Updated.